tests/instancetype/instancetype.go

```diff
@@ -1422,9 +1429,19 @@ var _ = Describe("[crit:medium][vendor:cnv-qe@redhat.com][level:component][sig-c
 
 	Context("VirtualMachine using preference resource requirements", func() {
 		var (
-			preferCores   = instancetypev1beta1.Cores
-			preferThreads = instancetypev1beta1.Threads
-			preferAny     = instancetypev1beta1.Any
+			preferCores    = instancetypev1beta1.Cores
+			preferThreads  = instancetypev1beta1.Threads
+			preferAny      = instancetypev1beta1.Any
+			guestMemory2GB = resource.MustParse("2GiB")
+			guestMemory1GB = resource.MustParse("1GiB")
```

Comment:
@anishbista60 the tests fail due to a panic here:
```suggestion
			guestMemory2GB = resource.MustParse("2Gi")
			guestMemory1GB = resource.MustParse("1Gi")
```